packages/notifi-dapp-example/src/components/TopicStackRow.tsx

```diff
@@ -94,7 +135,7 @@ export const TopicStackRow: React.FC<TopicStackRowProps> = (props) => {
               setIsTopicStackRowInputVisible(true);
             }}
           >
-            + Add Pair
+            {isTradingPairAlert ? '+ Add Pair' : '+ Add Fee Rate'}
```

Comment:
Similar comment as above, an approach is to use regex to resolve the `ref` and rendered the resolved value.